### ISSUE TITLE
AWS: bbl up --lb-chain flag removed in latest release

### ISSUE THREAD

**Issue by davewalter** [cloudfoundry/bosh-bootloader#525]
We have noticed a problem with v8.4.43 for our AWS environment that we use to test CF-Deployment. We have a job that runs `bbl plan`/`bbl up` for all of our CI test environments on a weekly basis, and it failed this morning with the following error:

```
$ bbl plan --debug --lb-type=cf --lb-cert=/tmp/bbl-cert --lb-chain=/tmp/bbl-cert-chain --lb-key=/tmp/bbl-key --lb-domain=example.com

flag provided but not defined: -lb-chain
```

I think that the new version has accidentally introduced some changes that were made shortly before the Infrastructure team stopped supporting BBL. Most of the changes appear to be desirable, but one of them was the [removal](https://github.com/cloudfoundry/bosh-bootloader/commit/61be38e13af18034bfc650968539240b3c7aa59e) of the `--lb-chain` flag from the `bbl up` command, which was introduced when PR #498 was merged. From what I can tell, this PR was related to issue #422, which is concerned with changes related to a switch to AWS network load balancers. However, the change in LB type on AWS was ultimately reverted [here](https://github.com/cloudfoundry/bosh-bootloader/commit/7d863730f1f2ccac66cbf12ffe9183def120ef1b), and so I do not think that the PR to remove the `--lb-chain` flag should have been merged.

Would it be possible to revert this change and cut a new release? I can submit a PR if desired.

**Comment by gbandres98** [comment]
We've published [v8.8.44](https://github.com/cloudfoundry/bosh-bootloader/releases/tag/v8.4.44), that reverts those changes, could you check that the flag works as intended? Thanks

**Comment by davewalter** [comment]
Thanks @gbandres98. That looks to have fixed the problem.
